### PR TITLE
refactor: remove redundant code in session logs output

### PR DESCRIPTION
**Pull request by RonenMars**
## Linear Ticket
https://linear.app/autokitteh/issue/UI-768/remove-redundant-code-in-session-logs-output

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [x] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/store/cache/useOutputsCacheStore.ts [hunk 1]
```diff
@@ -22,9 +22,9 @@ const createOutputsStore: StateCreator<OutputsStore> = (set, get) => ({
 			},
 		})),
 
-	reload: (sessionId: string) => {
+	reload: (sessionId: string, pageSize: number) => {
```
Comment (J1za) [inline src/store/cache/useOutputsCacheStore.ts:25]: `reload: (sessionId, pageSize) => {`

File: src/hooks/useVirtualizedList.tsx [hunk 2]
```diff
@@ -27,11 +27,7 @@ export function useVirtualizedList<T extends SessionOutput | SessionActivity>(
 	const store = type === SessionLogType.Output ? outputsCacheStore : activitiesCacheStore;
```
Comment (J1za) [inline src/hooks/useVirtualizedList.tsx:27]: rewrite as `const { loadLogs, loading, sessions } = type === SessionLogType.Output ? outputsCacheStore : activitiesCacheStore;
`

File: src/store/cache/useOutputsCacheStore.ts (hunk 1 above)
Comment (RonenMars) [reply src/store/cache/useOutputsCacheStore.ts]: refactored:
https://github.com/autokitteh/web-platform/pull/504/commits/1b4efefea1e0593562416649e43ab429e0739c3c

File: src/hooks/useVirtualizedList.tsx [hunk 3]
```diff
@@ -42,7 +38,14 @@ export function useVirtualizedList<T extends SessionOutput | SessionActivity>(
...
 
-	const { loadLogs, loading, reset } = store;
+	useEffect(() => {
+		if (!sessionId) return;
+
+		setSession(store.sessions[sessionId]);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [store.sessions]);
+
+	const { loadLogs, loading } = store;
```
Comment (J1za) [inline src/hooks/useVirtualizedList.tsx:48]: You can remove it

Comment (RonenMars) [reply src/hooks/useVirtualizedList.tsx]: refactored

File: src/hooks/useVirtualizedList.tsx (hunk 2 above)
Comment (RonenMars) [reply src/hooks/useVirtualizedList.tsx]: refactored

**Review by J1za — APPROVED** [review]

**Pull request merged**